appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '{build}'
+
+image: Visual Studio 2017
+
+# Test against the latest version of this Node.js version
+# environment:
+#   nodejs_version: "6"
+
+platform:
+  - x64
+
+cache:
+  - node_modules
+  - '%APPDATA%\npm-cache'
+
+init:
+  - git config --global core.autocrlf input
+
+# Install scripts (runs after repo cloning)
+install:
+  - ps: Install-Product node 6 x64
+  # - git reset --hard HEAD
+  - npm install
```

Comment:
Virtually all of the examples I’ve seen configure `npm install` under the `install` section, e.g., electron/electron-rebuild (https://github.com/electron/electron-rebuild/blob/master/.appveyor.yml), paulcbetts/spawn-rx (https://github.com/paulcbetts/spawn-rx/blob/master/appveyor.yml), and sqlectron/sqlectron-gui (https://github.com/sqlectron/sqlectron-gui/blob/master/appveyor.yml), as does AppVeyor in its section “Testing with Node.js and io.js” (https://www.appveyor.com/docs/lang/nodejs-iojs/).